docs/pages/access-controls/guides/guide-to-cluster-wide-idp-hardening.mdx

```diff
@@ -0,0 +1,660 @@
+---
+title: Guide to hardening against IdP compromises
+description: Implement cluster-wide hardening measures.
+---
+
+Protecting your organization against Identity Provider (IdP) compromises is more critical than ever. 
+This guide aims to help you fortify your identity infrastructure and mitigate the risks associated with IdP weaknesses.
+
+## Understanding IdP compromise
+
+An IdP compromise occurs when an attacker gains unauthorized access to your identity management 
+system, potentially allowing them to impersonate legitimate users, escalate privileges, or access 
+sensitive information. This can happen through various means, such as exploiting software vulnerabilities, 
+stealing credentials, or social engineering attacks.
+
+## Current security measures may fall short
+
+While many organizations have implemented basic security measures like single sign-on (SSO) and multi-factor authentication (MFA), 
+these alone may not be sufficient to protect against sophisticated attacks targeting your IdP. 
+Attackers are constantly evolving their techniques, and traditional security measures may have limitations 
+or vulnerabilities that can be exploited.
+
+## Hardening measures
```

Comment:
I think we can use the same structure we use for other how-to guides for this one:
- Remove this heading
- Restructure all remaining headings to have the structure:

  ```
   ## Prerequisites
   ## Step 1/n .
   ...
   ## Step n/n .
   ## Next steps
   ```

Reply:
Deleted